metabase/metabase
# Support Y-axis split

### Description

Supports Y-axis series settings and auto-split Y-axis logic. The code logic is copied from the old line/area/bar and adjusted to the new chart model, the old version will be removed with the dynamic charts.

The diff is huge due to added combo chart story and its sample data.

### How to verify

1. Create multiple line/area/bar/combo charts
    - Multiple metrics without breakout with big differences in Y-extent without modifying the Y-axis setting manually
    - A chart with breakouts without modifying the Y-axis setting manually
    - Same charts but with explicitly selected Y-axis for some series in different combinations
3. Send a test dashboard subscription with the charts
4. Ensure the Y-axis split is exactly the same

### Checklist
